monix-reactive/shared/src/main/scala/monix/reactive/subjects/Var.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2014-2018 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.subjects
+
+import monix.execution.{ Ack, Cancelable, Scheduler }
+import monix.reactive.{ Observable, OverflowStrategy }
+import monix.reactive.observers.Subscriber
+
+/** `Var` when subscribed, will emit the most recently emitted item by the source,
+  * or the `initialValue` (as the seed) in case no value has yet been emitted, then continuing
+  * to emit events subsequent to the time of invocation via an underlying [[ConcurrentSubject]].
+  *
+  * @see [[ConcurrentSubject]]
```

Comment:
I want a more detailed ScalaDoc with a sample preferably.

I'm thinking the sample should be like:

```scala
val a = Var(0)
val b = Var(0)

// Sum that gets re-calculated "reactively"
val sum = Observable.combineLatest(a, b)(_ + _)

// Subscribes for updates
sum.foreach(println)

a := 4
// => 4
b := 5
// => 9
a := 10
// => 15
```

And it should include a mention that this is equivalent with a `ConcurrentSubject.behavior(Unbounded)` that caches the current value for immediate usage.